### ISSUE TITLE
Décalage de la carte lors de la sélection d'une catégorie

### ISSUE THREAD

**Issue by Jean-BaptisteC** [cartesapp/cartes#480]
Comment reproduire:
- Ouvrir ce lien https://cartes.app/#17/48.8617/2.347/15/40
- Sélectionner un type lieu dans le carrousel ou sélectionner une catégorie -> La carte se décale

**Comment by laem** [comment]
Oui, c'est pour préparer l'arrivée d'informations dans l'encart à gauche. C'est un effet secondaire de cette implémentation qui se base sur l'URL : s'il y a un paramètre dans l'url, alors la carte estime qu'il y aura du contenu dans l'encart, donc elle se décale. 

Il faudrait calculer plutôt s'il y a un contenu dans l'encart, mais c'est moins trivial. 

De toutes façon, il faudrait faire un affichage en liste des lieux d'une recherche. Donc le problème sera réglé.

**Comment by etienneJr** [comment]
c'est le comportement voulu, donc on peut dire que l'issue est résolue, n'est-ce pas ?
